test/run_test.go

```diff
@@ -355,7 +355,6 @@ func TestLineDirectiveProcessedFiles(t *testing.T) {
 func TestUnsafeOk(t *testing.T) {
 	testshared.NewRunnerBuilder(t).
 		WithNoConfig().
-		WithArgs("--enable-all").
```

Comment:
how does this change relate to PR?

Reply:
If I return, can I add `-D gofactory` to fix linter error `Use factory for unsafe.Pointer`.
Or is solution from comment is better?
https://github.com/golangci/golangci-lint/pull/4196#discussion_r1443966248

Reply:
this should be reverted.